cockroachcloud/cloud-org-sso.md

```diff
@@ -30,31 +30,50 @@ Cloud Organization SSO allows you to customize your SSO configuration to meet yo
 - You can disable any authentication method. To enforce a requirement to use SSO, you can enable only SSO authentication methods and disable password authentication. If you disable password authentication, passwords are not retained.
-- [Autoprovisioning](#autoprovisioning) optionally removes the need to invite members to your organization. Autoprovisioning is disabled by default for each SSO authentication method.
 - You can [restrict the email domains](configure-cloud-org-sso.html#allowed-email-domains) that are allowed to sign in using an SSO authentication method. By default, any email domain is allowed.
+- [SSO Autoprovisioning](#sso-autoprovisioning) can be enabled per SSO authentication method, and automatically creates a {{ site.data.products.db }} organization account when a member successfully authenticates using an SSO authentication method for the first time, with no invitation required.
+- [SCIM Autoprovisioning](#scim-autoprovisioning) automatically creates a {{ site.data.products.db }} organization account when a user is assigned to the SCIM application in your IdP that is connected to your {{ site.data.products.db }} organization.
 
 To enable and configure Cloud Organization SSO, refer to [Configure Cloud Organization SSO](configure-cloud-org-sso.html). The following sections provide more details about the features of Cloud Organization SSO.
 
-### Autoprovisioning
+{{site.data.alerts.callout_success}}
+If your organization includes members whose identity you don't manage, such as partners or consultants, you can leave password authentication enabled for those members, while instructing your internal users to sign in using SSO. Members who sign in using a password must be removed from your {{ site.data.products.db }} organization manually.
+{{site.data.alerts.end}}
 
-Autoprovisioning allows you to centralize management of your users in an IdP and removes the need to [invite users to your organization](console-access-management.html#invite-team-members-to-cockroachdb-cloud). When autoprovisioning is enabled, the first time a new user successfully signs in using the custom sign-in page, a {{ site.data.products.db }} account is automatically created for them and the user is assigned the [Developer role](console-access-management.html#developer) by default.
+### SSO autoprovisioning
 
-Together with [Allowed Email Domains](configure-cloud-org-sso.html#allowed-email-domains), autoprovisioning allows new users to get started as soon as they are provisioned in your IdP, without waiting for an invitation.
+SSO autoprovisioning is a self-service mechanism that creates a {{ site.data.products.db }} organization when a user successfully authenticates for the first time, without the need for an [invitation(console-access-management.html#invite-team-members-to-cockroachdb-cloud). SSO autoprovisioning is disabled by default, but can be enabled per SSO authentication method.
```

Comment:
@nickvigilante This is a false + that I think you already have a report on. It's aggressive that it's a Failure -- wouldn't that block merging?